shared/src/common/enums/validation/add-episode-validation-message.enum.ts

```diff
@@ -0,0 +1,13 @@
+import { AddEpisodeValidationRule } from './add-episode-validation-rule.enum';
```

Comment:
rename to `shared/src/common/enums/validation/episode-validation-message.enum.ts`

Reply:
fixed it in next commit